servers/src/main/java/tachyon/master/file/FileSystemMaster.java

```diff
@@ -642,21 +618,30 @@ public FileBlockInfo getFileBlockInfo(long fileId, int fileBlockIndex)
     }
   }
 
+  /**
+   * Returns all the {@link FileBlockInfo} of the given file. Called via RPC, and internal masters.
```

Comment:
What calls this? I thought RPC only called the id version of this, and only the web ui might call this?